GomokuServer/src/GomokuServer.Api/Hubs/GameHub.cs

```diff
@@ -63,16 +61,9 @@ public async Task MakeMove(MakeMoveClientMessage makeMoveMessage)
 		}
 	}
 
-	public async Task SendMessage(byte[] packedData)
-	{
-		var decodedMessage = MessagePackSerializer.Deserialize<ChatMessageClientMessage>(packedData);
-		Console.WriteLine($"SendMessage called with user: {decodedMessage.User}, message: {decodedMessage.Message}");
-		await Clients.All.SendAsync("ReceiveMessage", decodedMessage.User, decodedMessage.Message);
-	}
-
 	public async Task ReceiveMessage(string user, string message)
 	{
 		Console.WriteLine($"ReceiveMessage called with user: {user}, message: {message}");
-		await Clients.Caller.SendAsync(GameHubMethod.ReceiveMessage, user, message);
+		await Clients.All.SendAsync(GameHubMethod.ReceiveMessage, user, message);
```

Comment:
I don't think Clients.All is correct in this case. We don't want to send to all clients this. Only in group by game id. Let's send gameId to this method as well and broadcast messages only to clients in this game

Reply:
Yes, I was also thinking to attach to gameID each chat, kinda makes sense